catalog/ui/catalog-ui-search/webpack/config/base.js

```diff
@@ -7,6 +8,18 @@ var resolve = function (place) {
   return path.resolve(__dirname, '../../', place)
 };
 
+let commitHash = require('child_process')
```

Comment:
I don't think we need to require the child_process module for every use.

Reply:
😂  oops, I want someone to test out this part on windows to make sure these work as well.